src/aiortc/codecs/vpx.py

```diff
@@ -271,8 +274,9 @@ def encode(
             self.cfg.g_h = frame.height
             self.cfg.rc_resize_allowed = 0
             self.cfg.rc_end_usage = lib.VPX_CBR
-            self.cfg.rc_min_quantizer = quantizer
-            self.cfg.rc_max_quantizer = quantizer
+            self.cfg.rc_min_quantizer = quantizer if quantizer > 0 else 0
```

Comment:
what defaults are you using to signal 0 - 63 range? can you clarify that?

Reply:
`-1`. added a note